packages/cozy-client/src/withMutations.jsx

```diff
@@ -1,7 +1,16 @@
+import merge from 'lodash/merge'
+import reduce from 'lodash/reduce'
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
-const withMutations = (mutations = {}) => WrappedComponent => {
+/**
+ * @function
+ * @description HOC to provide mutations to components. Needs client in context
+ * or as prop.
+ * @param  {Component} Component - wrapped component
```

Comment:
I think the interesting param to document here is `mutations` and not `WrappedComponent `, no?

Reply:
Oops you're totally right 👍